templates/taxi/manufacturer_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <form action="" method="post">
+    {% csrf_token %}
+    <h1>Delete manufacturer</h1>
+    <h3>Are you sure?</h3>
+    <input class="btn btn-danger" type="submit" value="Yes, delete">
+  </form>
+{% endblock %}
```

Comment:
Add Cancel button apart from Delete one. The Cancel button will lead to the previous page the user was on. Read checklist